prow/config/config.go

```diff
@@ -1357,6 +1358,10 @@ func ReadJobConfig(jobConfig string, yamlOpts ...yaml.JSONOpt) (JobConfig, error
 		return jc, nil
 	}
 
+	prowIgnore, err := gitignore.NewRepositoryWithFile(jobConfig, ".prowignore")
```

Comment:
nit: maybe `.prowignore` should be a constant, it's used from two places?

Reply:
Switched to a constant throughout.